README.md

```diff
@@ -1674,6 +1674,7 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 - [sovereign](https://github.com/sovereign/sovereign) - Set of Ansible playbooks to build and maintain your own private cloud: email, calendar, contacts, file sync, IRC bouncer, VPN, and more. `GPL-3.0` `YAML/Other`
 - [Syncloud](https://syncloud.org/) - Your own online file storage, social network or email server. ([Source Code](https://github.com/syncloud/platform)) `GPL-3.0` `Python/Other`
 - [UBOS](https://ubos.net/) - Linux distro that runs on indie boxes (personal servers and IoT devices). Single-command installation and management of apps - Jenkins, Mediawiki, Owncloud, WordPress, etc., and other features. `GPL-3.0` `Perl/Other`
+- [Urbit](https://urbit.org/) - Platform for decentralized network of personal servers, consisting of a pure single-function operating system (Arvo), an identity layer (Azimuth), and virtual machine (Vere). [Source Code](https://github.com/urbit/urbit) `MIT License` `Hoon/Nook/C` 
```

Comment:
Couple changes to be made:

- Your missing `(` and `)` around the source code link
-  `MIT License` should be shortened to `MIT`

>https://operators.urbit.org/manual/install

The installation docs also lead to a 404 so it may be better to wait for this to be resolved as users may need a hand with installation,

Reply:
@neocris ping can you please fix the points mentioned above?